packages/mobile/src/account/reducer.ts

```diff
@@ -64,11 +64,11 @@ export const reducer = (state: State | undefined = initialState, action: ActionT
         defaultCountryCode: action.countryCode,
       }
     case Actions.DEV_MODE_TRIGGER_CLICKED:
-      const newClickCount = (state.devModeClickCount + 1) % 10
+      const newClickCount = (state.devModeClickCount + 1) % 6
```

Comment:
nit: this seems a bit overkill to store the number of clicks in the redux store. Just a remark and unrelated to your changes of course ;)

Reply:
Yeah, I guess it could be a module-level variable instead.